frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-metrics/PipelineRunTaskRunGraph.tsx

```diff
@@ -115,43 +115,47 @@ const PipelineRunTaskRunGraph: React.FC<PipelineMetricsGraphProps> = ({
       onLegendClick: handleLegendClick,
     });
   return (
-    <>
-      <LineChart
-        ariaDesc={t('pipelines-plugin~Pipeline task run duration chart')}
-        ariaTitle={t('pipelines-plugin~Task run duration')}
-        data={_.values(finalObj)}
-        yTickFormatter={(seconds) => getYaxisValues(seconds)}
-        events={getEvents()}
-        hiddenSeries={hiddenSeries}
-        tickValues={tickValues}
-        width={width}
-        height={chartHeight}
-        legendPosition="bottom-left"
-        legendComponent={
-          <ChartLegend
-            gutter={25}
-            y={DEFAULT_CHART_HEIGHT + 75}
-            itemsPerRow={4}
-            name="legend"
-            data={getLegendData()}
-          />
-        }
-        containerComponent={
-          <ChartVoronoiContainer
-            constrainToVisibleArea
-            activateData={false}
-            voronoiPadding={{ bottom: 75 } as any}
-            labels={({ datum }) =>
-              datum.childName.includes('line-') && datum.y !== null
-                ? `${datum?.metric?.pipelinerun}
+    <Measure bounds>
+      {({ measureRef, contentRect }) => (
+        <div ref={measureRef}>
+          <LineChart
+            ariaDesc={t('pipelines-plugin~Pipeline task run duration chart')}
+            ariaTitle={t('pipelines-plugin~Task run duration')}
+            data={_.values(finalObj)}
+            yTickFormatter={(seconds) => getYaxisValues(seconds)}
+            events={getEvents()}
+            hiddenSeries={hiddenSeries}
+            tickValues={tickValues}
+            width={contentRect.bounds.width}
+            height={chartHeight}
```

Comment:
can we even get height from `contentRect.bounds.height`

Reply:
This is the last chart in the bottom row and this one has interactive legends under the chart, whereas the other chart in the bottom row doesn't have this legends. So fixed Height will make the cards even, if i use a dynamic height the cards wont be on the same size.